src/nvim/diff.c

```diff
@@ -1834,28 +1833,30 @@ int diffopt_changed(void)
   return OK;
 }
 
-/// Return TRUE if 'diffopt' contains "horizontal".
+/// Return true if 'diffopt' contains "horizontal".
 ///
-/// @return TRUE if 'diffopt' contains "horizontal"
-int diffopt_horizontal(void)
+/// @return true if 'diffopt' contains "horizontal"
```

Comment:
maybe @bfredl can chime in.  I think we should avoid such redundancy, but don't feel strongly on it.